src/env.d.ts

```diff
@@ -1,10 +0,0 @@
-/// <reference types="vite/client" />
```

Comment:
Any reason we are removing this? This is helpful to make env typesafe.

Reply:
I have moved the env typesafe file inside declarations folder.